README.md

```diff
@@ -165,5 +183,20 @@ relationships between the two are set up in Active Record.
 
 Write your code here:
 ```ruby
-# code here
+#1
+el_lamp = Lamp.create(wishes_remaining: 3)
+jeanie = Genie.crate(name: "Jean")
+
+#2
+el_lamp.update(genie_id: jeanie.id)
+
+#3
+el_lamp.update(wishes_remaining: 1)
+
+#4
+jafar = Genie.create(name: "Jafar", lamp_id: el_lamp_le_deux.id)
```

Comment:
You would need to define `el_lamp_le_deux` prior to this in order for this to work.